packages/aws-appsync-subscription-link/__tests__/link/realtime-subscription-handshake-link-test.ts

```diff
@@ -359,6 +362,164 @@ describe("RealTime subscription link", () => {
             }
 
         });
-    })
+    });
+
+    test("Can use a custom keepAliveTimeoutMs", (done) => {
+        const id = "abcd-efgh-ijkl-mnop";
+        uuid.mockImplementationOnce(() => id);
+
+        expect.assertions(5);
+        jest.spyOn(Date.prototype, 'toISOString').mockImplementation(jest.fn(() => {
+            return "2019-11-13T18:47:04.733Z";
+        }));
+        AppSyncRealTimeSubscriptionHandshakeLink.createWebSocket = jest.fn((url, protocol) => {
+            expect(url).toBe('wss://apikeytest.testcustomdomain.com/graphql/realtime?header=eyJob3N0IjoiYXBpa2V5dGVzdC50ZXN0Y3VzdG9tZG9tYWluLmNvbSIsIngtYW16LWRhdGUiOiIyMDE5MTExM1QxODQ3MDRaIiwieC1hcGkta2V5IjoieHh4eHgifQ==&payload=e30=');
+            expect(protocol).toBe('graphql-ws');
```

Comment:
Feels like we're reasserting things that are essentially duplicate tests. Any reason to have more expects in this outside of the timeout?

Reply:
Yeah, mostly to prove that the new config option doesn't mess with the auth